DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -464,21 +467,12 @@ static function AddNote(#var(PlayerPawn) player, bool bEmptyNotes, string text)
     }
 }
 
-function AddGoalFromConv(#var(PlayerPawn) player, name goaltag, name convname)
-{
-    local DeusExGoal newGoal;
-    newGoal = player.FindGoal(goaltag);
-    if(newGoal != None) return;
-    newGoal = player.AddGoal(goaltag, true);
-    if(newGoal == None) return;
-    newGoal.SetText(GetGoalTextGC(goaltag, convname));
-}
-
 function MarkConvPlayed(name flagname, bool bFemale)
```

Comment:
If you're never going to use the name as-is then no use having it added to the names table, better of keeping it a string until the last minute, which also means changing the single quotes to double quotes when calling this function

```suggestion
function MarkConvPlayed(string flagname, bool bFemale)
```